### DISCUSSION TITLE
Stuck at boot screen after 4.2.3 upgrade.

### DISCUSSION THREAD

**Discussion by alexboffey** [OnionUI/Onion#1351] [Support]
Have just installed 4.2.3 upgrade to SD card on MacOS. My device (MM+) was running fine on 4.2.0-beta and now is stuck on the miyoo boot screen after upgrade following the steps here: https://onionui.github.io/docs/installation/update?os=mac

Any suggestions on how to fix this, I'd like to not lose my game progress if possible.
Thanks

**Comment by thomasklemenc** [comment]
Are you using the SD card that came with the device? These are often the cause of such issues.
Can you still access the SD from your computer? First step would be to backup the "Saves" folder.

You can try a clean reinstall, delete everything but the folders Roms, Bios and Saves and start over from [here](https://onionui.github.io/docs/installation/fresh?os=mac)

**Reply by alexboffey** [reply]
Yes it's the SD that came with the device. Will try this, thanks for your help!
